src/AEG/AEGTree.ts

```diff
@@ -149,6 +146,24 @@ export class AEGTree {
         return this.internalSheet.getLowestParent(incomingPoint);
     }
 
+    public getParent(incomingNode: CutNode | AtomNode): CutNode | null {
```

Comment:
Documentation plz

Reply:
You know what's so funny this is no longer used because we don't allow adoption lmao I forgot to delete this